CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# v1.33.0 (2025-02-12)
+
+## Release Highlights
+* Remove aws-k8s-1.24 variants (https://github.com/bottlerocket-os/bottlerocket/issues/4359)
+* Add support for NVIDIA Multi-Instance GPU (MIG) (https://github.com/bottlerocket-os/bottlerocket/issues/4252)
+
```

Comment:
Can we change these to be numerical links in the same way that we do for pull requests?

Reply:
Should I do it this way `[#4359](https://github.com/bottlerocket-os/bottlerocket/issues/4359)`? That's how we did in the bottom but there were other ways too like just the plain link.

Reply:
ohh looks like those numbers are unique and there is no overlap between PR and issues. I will do it similar to PRs and add the links at the bottom.